src/Authentication/Authentication.Core/Utilities/AuthenticationHelpers.cs

```diff
@@ -86,6 +88,12 @@ private static bool IsWamSupported()
             return GraphSession.Instance.GraphOption.EnableWAMForMSGraph && SharedUtilities.IsWindowsPlatform();
         }
 
+        //Check to see if ATPoP is Supported
+        public static bool IsATPoPSupported()
```

Comment:
I don't think this is correct. AT POP for public client (!) is supported when: 

- broker is available and enabled (currently this means Windows except Win Sever 2016 + WAM enabled)
- service supports AT POP